frontend/src/apiMocks/mockData.ts

```diff
@@ -228,3 +236,28 @@ export const domainaddresses: Record<
     },
   ],
 };
+
+export const mockedRealphones: Record<typeof mockIds[number], RealPhoneData> = {
+  empty: [],
+  onboarding: [],
+  some: [
+    {
+      id: 0,
```

Comment:
`id`s should never be 0.

Reply:
Is that problematic in mocks? I'm not particularly opposed to changing it, but it does complicate the `POST` mocks a bit (right now I can just use the number of existing mocks as the new ID).

(Actually, I'd be interested in learning why that's problematic in actual back-ends too, but only if you have time :smile: )

Reply:
Oh ... it's not a technical problem - more of a "technically impossible" problem. It should never BE ABLE to be 0, since it's an auto-increment integer id field. So less of a "don't do this" and more of a "the DB will never do this."